src/main/java/uk/gov/companieshouse/api/accounts/model/rest/smallfull/notes/relatedpartytransactions/RptTransaction.java

```diff
@@ -3,22 +3,27 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
+import javax.validation.constraints.NotNull;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class RptTransaction extends RestObject {
 
     @JsonProperty("name_of_related_party")
     private String nameOfRelatedParty;
 
+    @NotNull
     @JsonProperty("relationship")
     private String relationship;
 
+    @NotNull
     @JsonProperty("description_of_transaction")
     private String description;
```

Comment:
will this cause issues with de-serialising? usually we match the name of the json property to the java variable just in camel case, can we review this?